README.md

```diff
@@ -1,13 +1,23 @@
 [![Standard - JavaScript Style Guide](https://cdn.rawgit.com/feross/standard/master/badge.svg)](https://github.com/feross/standard)
 [![styled with prettier](https://img.shields.io/badge/styled_with-prettier-ff69b4.svg)](https://github.com/prettier/prettier)
 
-## Linklet
+# Linklet
+
+Linklet contains the links which are shared in whatsapp freeCodeCamp Hyderabad group. This app makes it easier to search and find usefull links share in the group.
+
+# Linklet Android App
+A native android app with the same functionality of web app. Its data source is based on [Linklet's REST API](https://github.com/vinaypuppal/linklet-api).
```

Comment:
I thinks its best to have just link to your repo's in this Readme and add this info to your repos Readme because this repo does not include android app.